mvp-spec/dsp-implementation.md

```diff
@@ -196,138 +166,62 @@ are subject to adaptation to the context. For instance, if the
-<!--partial-end-->
-
-#### The Transmission Result object
-
-<!--partial-begin { "files": [ "transmission-result-table.md" ] } -->
-<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
-| Field           | Type                          | Details                           |
-|-----------------|-------------------------------|-----------------------------------|
-| version         | Number                        | The version of the PAF used.                                                                                                                                                                                                                               |
-| receiver        | String                        | The domain name of the DSP.                                                                                                                                                                                                                                                                                |
-| status          | String                        | Equals "success" if the DSP signed the Transmission and returns it to the sender.<br /> Equals "error_bad_request" if the receiver doesn't understand or see inconsistency in the Transmission Request.<br /> Equals "error_cannot_process" if the receiver failed to use the data of the Transmission Request properly. |
-| details         | String                        | In case of an error status, the DSP can provide details concerning the error.                                                                                                                                                                                                                              |
-| source          | Source object                 | The source contains all the data for identifying the DSP and verifying the Transmission.                                                                                                                                                                                                                   |
-<!--partial-end-->
+## The Identity endpoint
 
-#### Signatures
+The Contracting Party must expose an Identity endpoint. It gathers information for verifying the identity and the transmissions that it signed.
 
-Note that the signature of the `source` in the Transmission Request is 
-generated with the following string:
-
-<!--partial-begin { "files": [ "transmission-request-signature-string.txt" ], "block": "" } -->
-<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+It is reachable at the following endpoint:
 ```
-transmission_request_receiver_domain        + '\u2063' +
-transmission_request.source.domain          + '\u2063' + 
-transmission_request.source.timestamp       + '\u2063' + 
-seed.source.signature
+GET https://<domain>/prebid/v1/identity
```

Comment:
I think we said we use `/paf/v1/identity`. (this is what I have been using in the rest of the repo).

Reply:
Fixed